### PR TITLE
Restoring ability to toggle noise generation

### PR DESCRIPTION
**Pull request by wforeman**
Just a couple very minor updates. Restored the function of the "GenNoise" boolean in the simwire configuration, so users can turn on/off noise for tests. Also skipping the pedestal RMS routine if this is set to 0 in the fcl.

### CODE REVIEW

**Comment by mstancar** [comment]
The noise service already provides this option??  I have no objection if this is not possible through the service

**Comment by wforeman** [comment]
Noise service options (DetectorSim/Services/noiseservices_sbnd.fcl) are rather complicated. Best I can tell, looks like they allow you to enable different individual types of noise like white noise, gaussian, coherent, etc. If I wanted to turn off the noise altogether, it's unclear what I should be changing in the noise service. So I think there's value in having a single module-level parameter that decides whether the noise service is called at all.

**Comment by mstancar** [comment]
Agreed, just add comment to the fcl file then that specify that when this is set, the noise service is not called.

**Review by mstancar — APPROVED** [review]
Please also add a comment to the fcl file specifying that this parameter when set to false, tells the code not to call the noise service.

**Comment by pgreen135** [comment]
trigger build

File: sbndcode/DetectorSim/detsimmodules_sbnd.fcl [hunk 1]
```diff
@@ -12,9 +12,9 @@ sbnd_simwire:
  NoiseRand:           0.1          #frac of randomness of noise freq-spec
  LowCutoff:           7.5          #Low frequency filter cutoff (kHz)
  CompressionType:     "none"       #could also be none		
- GenNoise:            true        #gen noise...if false function not called
- GetNoiseFromHisto:   false
- GenNoiseInTime:      true
+ GenNoise:            true        # If false, NoiseService function is not called
+ GetNoiseFromHisto:   false       # NO LONGER USED; set through NoiseService instead
+ GenNoiseInTime:      true        # NO LONGER USED; set through NoiseService instead
```
Comment (absolution1) [inline sbndcode/DetectorSim/detsimmodules_sbnd.fcl:17]: Could these be removed if they are no longer used?

Comment (wforeman) [reply sbndcode/DetectorSim/detsimmodules_sbnd.fcl:17]: I was going to remove them, but didn't want to change too much at once. We could certainly remove "GenNoiseInTime" with no consequences, but "GetNoiseFromHisto" still appears in an uncommented code block where the noise spectrum histogram is retrieved... though this histogram is not used at all anywhere else.

Comment (absolution1) [reply sbndcode/DetectorSim/detsimmodules_sbnd.fcl:17]: I'd vote for cutting them now.  They still exist in history so they can be reimplemented if needs be.

Comment (wforeman) [reply sbndcode/DetectorSim/detsimmodules_sbnd.fcl]: Done!

**Comment by pgreen135** [comment]
trigger build

**Comment by pgreen135** [comment]
@wforeman looks like there's a c7 build error from an unused variable -- see: https://dbweb8.fnal.gov:8443/LarCI/app/ns:SBND/build_detail/phase_details?build_id=sbnd_ci/6518&platform=Linux%203.10.0-1160.21.1.el7.x86_64&phase=build&buildtype=slf7%20c7:prof

**Comment by wforeman** [comment]
@pgreen135, fixed. The variable (fSampleRate) was also technically unused before as well, but the compiler counts a variable as "used" even if it appears in a commented-out block... weird!

**Comment by pgreen135** [comment]
that is very strange! thanks for fixing it promptly

**Pull request merged**